.github/workflows/cache-new-image.yml

```diff
@@ -14,9 +14,9 @@ jobs:
       contents: read
       packages: write
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Log in to the Container registry
-        uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
+        uses: docker/login-action@v2
```

Comment:
Do we have any sane and reasonable way about getting notified when these actions release a new version that should be evaluated for updating?

Reply:
There might be a way, but right now we know b/c the publisher will release a warning that is visible when the action runs (like when it runs in _our_ actions).

Reply:
we're in the same boat here as other external api's, libraries, etc.  we can only hope they do a reasonable job on their side over time, use semver appropriately, and allow us to focus on the next thing :) otherwise we run off to something better or just build our own widget.

we also need to pay attention to what is available at https://github.com/actions and refactor things as appropriate from time to time.